recipe/meta.yaml

```diff
@@ -16,19 +16,25 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
     - pip
+    - setuptools >=61.2
+    - wheel
+    - setuptools_scm >=3.4.3
+    # From setuptools_scm[toml]
+    - tomli >=1
   run:
-    - python >=3.8
+    - python >={{ python_min }}
     - typing_extensions
 
 test:
   imports:
     - bytecode
-    - bytecode.instr
-    - bytecode.flags
-    - bytecode.concrete
-    - bytecode.cfg
+  commands:
+    - pip check
```

Comment:
Could you quickly explain those changes ?

Reply:
Just following the usual patterns with the test section adding `pip check` for dependency checking. We can add back to sub-package import test but in my experience those tend to change breaking the test in the future

Reply:
See: https://conda-forge.org/docs/maintainer/adding_pkgs/#test

Reply:
@MatthieuDartiailh anything else needed here?